ios/Sources/GutenbergKit/Sources/EditorAssetsLibrary.swift

```diff
@@ -0,0 +1,255 @@
+            endpoint = apiRoot.appendingPathComponent("wpcom/v2/editor-assets")
+        } else {
+            throw ManifestError.unavailable
+        }
+
+        // TODO: Invalid cache after a certain duration
+        let fileManager = FileManager.default
+        let cached = assetsDirectory.appendingPathComponent("editor-assets.json")
+        if !ignoreCache, fileManager.fileExists(atPath: cached.path) {
+            return try Data(contentsOf: cached)
+        }
+
+        var request = URLRequest(url: endpoint)
+        request.setValue(configuration.authHeader, forHTTPHeaderField: "Authorization")
+        let (data, response) = try await URLSession.shared.data(for: request)
+        if let status = (response as? HTTPURLResponse)?.statusCode, (200..<300).contains(status) {
+            if !fileManager.fileExists(atPath: cached.deletingLastPathComponent().path) {
+                try fileManager.createDirectory(at: cached.deletingLastPathComponent(), withIntermediateDirectories: true)
+            }
+
+            try data.write(to: cached)
+            return data
+        } else {
+            throw ManifestError.invalidServerResponse
+        }
+    }
+
+    /// Returns the `EditorConfiguration.editorAssetsEndpoint` manifest content, with JavaScript and stylesheet links
+    /// modified so that their content can be cached and reused by the editor.
+    ///
+    /// - SeeAlso: `CachedAssetSchemeHandler`
+    /// - SeeAlso: `EditorAssetsLibrary.addAsset`
+    func manifestContentForEditor() async throws -> Data {
+        let data = try await loadManifestContent()
+        let manifest = try JSONDecoder().decode(EditorAssetsMainifest.self, from: data)
+        return try manifest.renderForEditor()
+    }
+
+    /// Fetches all assets in the `EditorConfiguration.editorAssetsEndpoint` manifest and stores them on the device.
+    ///
+    /// - SeeAlso: CachedAssetSchemeHandler
+    public func fetchAssets() async throws {
+        let data = try await loadManifestContent(ignoreCache: true)
+        let manifest = try JSONDecoder().decode(EditorAssetsMainifest.self, from: data)
+        let assetLinks = try manifest.parseAssetLinks()
+
+        for link in assetLinks {
+            guard link.hasPrefix("http://") || link.hasPrefix("https://") else {
+                NSLog("Unexpected asset link: \(link)")
+                continue
+            }
+
+            guard let url = URL(string: link) else {
+                NSLog("Malformed asset link: \(link)")
+                continue
+            }
+
+            _ = try await cacheAsset(from: url, ignoreExistingCache: true)
+        }
+        NSLog("\(assetLinks.count) resources processed.")
+    }
+
+    /// Fetches one asset (JavaScript or stylesheet) and caches its content on the device.
+    func cacheAsset(from httpURL: URL, ignoreExistingCache: Bool = false) async throws -> URL {
```

Comment:
Reminder to check if https://github.com/wordpress-mobile/GutenbergKit/commit/590659ee4b7d2f0eca77742c676b77cb035d4cd7 is needed.

@dcalhoun Do you remember why these changes are necessary, and how to get those links to appear in the `wpcom/v2/editor-assets` endpoint?

https://github.com/wordpress-mobile/GutenbergKit/blob/590659ee4b7d2f0eca77742c676b77cb035d4cd7/ios/Sources/GutenbergKit/Sources/EditorLibrary.swift#L229-L231

Reply:
The need originated from encoding differences in asset URLs between the `scripts`/`styles` and `scripts_html`/`styles_html` keys included in https://github.com/Automattic/jetpack/pull/44046. If we no longer need the arrays of script/style URLs, then we no longer need to handle these encoding differences. 

To answer your question directly, I believe specific links causing issues originated from Jetpack stylesheets, but I do not recall which ones.